### PR TITLE
Fix form extensions for editable option

### PR DESCRIPTION
**Pull request by rn0**

### CODE REVIEW

File: lib/FSi/Component/DataGrid/Extension/Symfony/ColumnTypeExtension/FormExtension.php [hunk 1]
```diff
@@ -221,30 +223,21 @@ private function createForm(ColumnTypeInterface $column, $index, $data)
...
         $formBuilder = $this->formFactory->createNamedBuilder(
             $this->formName,
             'collection',
-            array($index => $dataArray),
+            array($index => $data),
             array(
                 'type' => new RowType($fields),
                 'csrf_protection' => false,
+                'options' => array(
+                    'data_class' => ClassUtils::getRealClass(get_class($data))
```
Comment (rn0) [inline lib/FSi/Component/DataGrid/Extension/Symfony/ColumnTypeExtension/FormExtension.php]: `$data` should be object. Add check

**Comment by chives** [comment]
:+1:

**Comment by bartekgruszka** [comment]
:+1:

**Comment by JarekW** [comment]
:+1:

**Pull request merged**